packages/@aws-cdk/cloudformation-include/test/valid-templates.test.ts

```diff
@@ -240,6 +240,33 @@ describe('CDK Include', () => {
     );
   });
 
+  test('paramters shadowed my expression in the map of Fn::Sub are not replaced', () => {
```

Comment:
1. I feel like this should be grouped with other tests that pass `parameters`, not with `Fn::Sub` tests.
2. The description is weird... I don't know what exactly it's saying. How about:

```suggestion
  test('does not modify Fn::Sub variables shadowing a replaced parameter', () => {
```